src-java/testing/test-library/src/main/java/org/openkilda/testing/service/tsdb/model/StatsResponseDto.java

```diff
@@ -13,15 +13,33 @@
  *   limitations under the License.
  */
 
-package org.openkilda.testing.service.otsdb.model;
+package org.openkilda.testing.service.tsdb.model;
 
-import java.util.Collections;
+import lombok.Data;
 
-public class EmptyStatsResult extends StatsResult {
-    public EmptyStatsResult() {
-        super();
-        tags = Collections.emptyMap();
-        aggregateTags = Collections.emptyList();
-        dps = Collections.emptyMap();
+import java.util.HashMap;
+import java.util.List;
+
+@Data
```

Comment:
Here and in other places fields have a default access modifier, but a private is more appropriate for a 
DTO. You can use `@FieldDefaults(makeFinal=true, level=AccessLevel.PRIVATE)` on a class level.
Or is there a reason not to do so?